src/MonthView/phone/ScrollListTaskComponent/ScrollListTaskComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react"
+import { Text, TouchableOpacity, View } from "react-native"
+import styles from "./ScrollListTaskComponent.styles"
+
+interface ComponentProps {
```

Comment:
I am thinking maybe we dont need to give width. Maybe just use `flex: 1`

Imagine you are creating the schedule list. Maybe you wont even know exactly the width of the column or you dont need to know.

You just need to care if you put 3 of them, it will divide it nicely

Reply:
nice!